### PR TITLE
🎉Add a usage message to the template

### PR DESCRIPTION
**Pull request by hairmare**
This adds an optionally overridable usage message template to all the documents. There is also docs on how to override the default usage section in README.md.

@paraenggu Let me know if there are any files I'm updating that need to have a custom message and I'll fix them up.

### CODE REVIEW

**Comment by paraenggu** [comment]
> @paraenggu Would you have any specific default text for snmp in mind, it seems like most of those need to be manually taken care of anyway (ie. the AVT one).

There's a default text within the [current documentation for SNMP](https://github.com/radiorabe/rabe-zabbix/tree/bb6acbdeb4ccfdd616ff63fab47db5698f55a43a#adding-an-snmp-template), but it requires modifications for each template. So you could either exclude the SNMP template from the default usage file generation, or generate an override containing the current README's SNMP usage with a note, that users must update the usage override in every case.

File: update-app-doc.xsl [hunk 1]
```diff
@@ -317,7 +348,7 @@ the terms of the GNU Affero General Public License as published by the Free
 Software Foundation, version 3 of the License.
 
 ## Copyright
-Copyright (c) 2017 - 2018 [Radio Bern RaBe](http://www.rabe.ch)
+Copyright (c) 2017 - 2019 [Radio Bern RaBe](http://www.rabe.ch)
```
Comment (paraenggu) [inline update-app-doc.xsl]: Can we insert the current year automatically instead of having it hard-coded?

Comment (hairmare) [reply update-app-doc.xsl]: Actually, that this is much earlier (ie. before the # Usage part) is very much intended.

File: ipmi/Threshold_Sensors/README.md [hunk 2]
```diff
@@ -9,11 +9,16 @@ Triggers will be created for each discovered sensor, according to their
 advertised lower and upper non-critical, critical and non-recoverable
 thresholds.
 
+This template is part of [RaBe's Zabbix template and helpers
```
[the remark this answers is not in the document]
Comment (hairmare) [reply ipmi/Threshold_Sensors/README.md]: Actually, that this is much earlier (ie. before the # Usage part) is very much intended.

**Comment by hairmare** [comment]
> There's a default text within the current documentation for SNMP, but it requires modifications for each template. So you could either exclude the SNMP template from the default usage file generation, or generate an override containing the current README's SNMP usage with a note, that users must update the usage override in every case.

The default stuff from the README.md now generates something sensible for snmp. I think this is a "feel free to improve post merge" now.

File: update-app-doc.xsl (hunk 1 above)
Comment (paraenggu) [reply update-app-doc.xsl]: Addressed with cd53272cb442a5636834e478facc0395be82d293

**Pull request merged**